docs/_blog/2020-04-16-working-with-private-registries.md

```diff
@@ -0,0 +1,139 @@
+
+### Working with Private Registries in Codewind
+
+In this blog entry let's explore working with private NPM registries and private Maven repositories in Codewind. Most developers should already be familiar with how to configure their environment to access private servers, but if you need a refresher, take a look at the [NPM documentation](https://docs.npmjs.com/configuring-npm/npmrc.html) or [Maven documentation](https://maven.apache.org/settings.html#Servers) on the steps involved. If you happen to be building your application by Docker (and who isn't these days?), then there is an added layer of complexity with the setup, such as making the configuration available to the Docker build while ensuring you do not leak any sensitive data--your server username and password--in the final Docker image. Fortunately, Codewind provides mechanisms and project templates that makes this process simpler.
+
+### A Node.js NPM Example
+
+I'm starting with a Codewind project created using the Node.js Express project template. When I open the application and navigate to its `/codewind` endpoint, I'm greeted with:
+
+![image of Hey there, this is codewind saying hi and thanks for using me](images/blog/privateregnode01.png){:width="500px"}
+*Hey there, this is codewind saying hi and thanks for using me*
+{:style="margin: 30px; text-align: center;"}
+
+The message looks pretty bland doesn't it? Luckily, I happen to have a special package hosted on my private GitHub NPM registry that I can use to give this message some flair. First, I'll configure the credentials for the registry in my `~/.npmrc` configuration file:
+
+```properties
+@makandre:registry=https://npm.pkg.github.com/makandre
+//npm.pkg.github.com/:_authToken=****************************************
+```
+{:style="margin: 30px;"}
+
+Now I can install the package to my project:
+
+```bash
+cwNodeProj$ npm install @makandre/samplepkg
+```
+{:style="margin: 30px;"}
+
+And now to use the package in the `cwNodeProj/server/routers/codewind.js` file:
+
+```js
+const express = require('express');
+const hello = require('@makandre/samplepkg'); // added require here
+
+module.exports = function (app) {
+  const router = express.Router();
+
+  router.get('/', function (req, res, next) {
+    // replaced "Hey there" with a call to hello()
+    const stringToReturn = hello() + ", this is codewind saying hi and thanks for using me"
+    res.status(200).send(stringToReturn);
+  });
+
+  app.use('/codewind', router);
+}
+```
+{:style="margin: 30px;"}
+
+All is set so I force a full project build in Codewind--right-click project in Codewind view and select *Build*... and the build fails. I check the build log and see the following:
```

Comment:
- It might look a bit clearer if you end with a period after "Codewind" and start a new sentence with "right-click".
"...in Codewind. Right-click the project in the Codewind view..."
- No space needed around `...`
- Docs use bold for elements like **Build**, but not a hard and fast rule to apply the same thing to blogs.

Reply:
I reworded the entire sentence here, hope that is better. I changed the **Build** to bold and it felt too strong imo, if it isn't a hard rule I'd like to leave it as *Build*